### ISSUE TITLE
MicroSD not accesible in LAN-only mode

### ISSUE THREAD

**Issue by zlice** [bambulab/BambuStudio#3210]
Using BambuStudio on Linux (1.8.2) says the microSD can't be accessed in LAN-only mode. I imagine it's just a file protocol over wifi and easy enough to view files. I was trying to delete files since I keep printing test parts and I don't need them anymore on the SD card.

![bambux1-microsd-lan](https://github.com/bambulab/BambuStudio/assets/8009811/8f96837f-99f4-4cbf-bf69-23293b7108b9)

**Comment by bambu123** [comment]
> Same problem here... The latest P1P/P1S update stated the following: "Support SD card remote browsing for time-lapses and models"
> 
> Does this only apply when using the cloud? If so, this should be made clear in the release notes and please be released for the lan-mode as well. Thanks in advance!

Yes, LAN-Only mode file protocol is not support for all printer models until now.

**Comment by ian-morgan99** [comment]
Same problem on a brand new A1 currently. I would really like to be able to access the SD card in lan only mode.
This could be done via FTPS over port 990, as I have confirmed manually today.

**Comment by fazadaoc** [comment]
Was this ever reconsidered? The fact that one can't manage freely files on the sd and start prone jobs from it is quite absurd
